### PR TITLE
Add banner and content for USS 2023

### PR DESCRIPTION
**Pull request by SenonETS**
Added banners, enable page uss23, with schedule page to be modified, with no speakers, no posters
@nandahkrishna Please take a look :).

### CODE REVIEW

File: _pages/2023/uss.md [hunk 1]
```diff
@@ -6,23 +6,22 @@ permalink: /2023/uss
 
 <br>
 
-# The UNIQUE Student Symposium 2023 is coming soon!
```
Comment (nandahkrishna) [inline _pages/2023/uss.md:9]: This page still has a lot of content from last year and should be updated before we merge the PR.

Comment (SenonETS) [reply _pages/2023/uss.md:9]: Is it about the section "Important dates"?

Comment (nandahkrishna) [reply _pages/2023/uss.md:9]: Not just the dates but some of the descriptions as well, for example:
1. Have we decided whether there will be posters or lighting talks or both?
2. What is the event subtitle? It is still set to last year's (Dynamics of Learning).
3. The accepted posters page should not yet be linked.
4. Is the submission deadline the same this year as well? (May 27)

Comment (SenonETS) [reply _pages/2023/uss.md:9]: Added one more commit to fix the main page uss23:
1. changed the title
2. disabled accepted posters for now
3. fixed the important dates
4. in-person for posters and the lightnighting talks

File: _pages/2023/uss.md [hunk 2]
```diff
@@ -6,48 +6,48 @@ permalink: /2023/uss
...
+##### Day 2: Tuesday, 6 June 2023
 ##### See the speakers [here](speakers).
 
 ---
 
 ## Important dates
 
-* Posters and Lightning Talks Registration due: <b>Friday, May 27th, 2022 (anywhere on Earth)</b>
-* Registration is closed. Thanks for joining us!
+* Posters and Lightning Talks Registration due: <b>Friday, May 7th, 2023 (in-person)</b>
```
Comment (nandahkrishna) [inline _pages/2023/uss.md]:
```suggestion
* Posters and Lightning Talks Registration due: <b>Friday, May 7th, 2023 (anywhere on Earth)</b>
```
This is about the time zone to submit the posters online. Are you sure that this is the correct deadline?

Comment (SenonETS) [reply _pages/2023/uss.md]: Yep

**Pull request merged**